tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Exporter" id="genomespace_exporter" require_login="True" version="0.0.4">
+<tool name="GenomeSpace Exporter" id="genomespace_exporter" version="0.0.4">
     <description> - send data to GenomeSpace</description>
-    <command interpreter="python">genomespace_exporter.py 
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
+    <command interpreter="python">genomespace_exporter.py
+        #set $target_folder = $genomespace_browser.split("^")[0]
+        #set $token = $genomespace_browser.split("^")[1] or $__user__.preferences.get( 'genomespace_token', None )
```

Comment:
Ah, that's unfortunate. I don't think this would not be a problem for pulsar, I believe the working directory (where the file ends up) would be staged. You could also try https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-environment-variables-environment-variable instead of writing out the token to a file.